examples/CRISP/interfold.config.yaml

```
@@ -5,7 +5,32 @@ chains:
     enabled: true
     rpc_url: ws://localhost:8545
     contracts:
+      e3_  - name: "mainnet"
+    enabled: false
```

YAMLlint (1.37.1)

[error] duplication of key "enabled" in mapping

(key-duplicates)